libs/market-depth/src/lib/orderbook-row.tsx

```diff
@@ -50,12 +50,12 @@ export const OrderbookRow = React.memo(
         <PriceCell
           testId={`price-${price}`}
           value={BigInt(price)}
-          valueFormatted={addDecimalsFormatNumber(price, decimalPlaces)}
+          valueFormatted={addDecimalsFormatNumber(price, positionDecimalPlaces)}
```

Comment:
chore (blocking): Should be market.decimalPlaces for price

Reply:
changed it back